io.openems.edge.evcs.goe.chargerhome/src/io/openems/edge/evcs/goe/chargerhome/Config.java

```diff
@@ -0,0 +1,35 @@
+package io.openems.edge.evcs.goe.chargerhome;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(name = "EVCS go-e Charger Home", //
+		description = "Implements the go-e Charger Home electric vehicle charging station.")
+@interface Config {
+
+	@AttributeDefinition(name = "Component-ID", description = "Unique ID of this Component")
+	String id() default "evcs0";
+
+	@AttributeDefinition(name = "Alias", description = "Human-readable name of this Component; defaults to Component-ID")
+	String alias() default "";
```

Comment:
Maybe set default to "Go-e Charger [COUNT OF CHARGERS]"

Reply:
This is really unusual in the Openems Project. In any other EVCS the default value is an empty string